app/walletcore/helper.go

```
@@ -87,3 +87,25 @@ func BuildTxDestinations(destinationAddresses []string, destinationAmounts []str
 	}
 	return
 }
+
+func WalletConnectionInfo(wallet Wallet, netType string) (*ConnectionInfo, error) {
+	accounts, err := wallet.AccountsOverview(DefaultRequiredConfirmations)
+	if err != nil {
+		return nil, fmt.Errorf("error fetching account balance: %s", err.Error())
+	}
+
+	var totalBalance Balance
+	for _, acc := range accounts {
+		totalBalance.Spendable += acc.Balance.Spendable
+		totalBalance.Total += acc.Balance.Total
+	}
+
+	bestBlock, _ := wallet.BestBlock()
```

Don't discard this error. Return it so it may be log appropriately.